fastapi/app/main.py

```
@@ -119,117 +56,154 @@ async def process_video(file: UploadFile = File(...)):
+                data={"model": "whisper-1"}
+            )
+
+            if response.status_code != 200:
+                logger.error(response.text)
+                raise HTTPException(500, "Whisper API 호출 실패")
+
+            result = response.json()
+            text = result.get("text", "")
+
+        # GPT 요약 생성 부분 (OpenAI API 호출)
+        generated_text = text
```

Ruff (0.11.9)

Local variable `generated_text` is assigned to but never used

Remove assignment to unused variable `generated_text`

(F841)